docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -69,6 +69,15 @@ NOTE: If no Events or Functions are specified, then the Sentinel will monitor AL
 
 IMPORTANT: At this time, Sentinels can only monitor internal function calls on Mainnet, Ropsten, and Kovan.  If the internal call emits an event, consider monitoring for that event instead.  Events emitted from internal calls are detected on all networks.
 
+[[hedera-support]]
+=== Hedera Support
+
+Hedera mainnet and testnet are supported in Defender Sentinels. There exist some considerations unique to this network when configuring a sentinel:
+
+* The sentinel monitors a contract's EVM address (starts with "0x") not the Hedera contract ID. The EVM address can be found on a Hedera Explorer such as [HashScan](https://hashscan.io/)
+* There is no remote resource to obtain contract ABIs as of now, so they must be generated from the solidity code using a [solidity compiler](https://docs.soliditylang.org/en/latest/installing-solidity.html).
```

Comment:
I'd rather redirect users to Remix or some ABI-generation specific docs. But this is a nitpick

Reply:
had the same thought but figured Remix is the top recommendation on that link so some users will head there anyhow, but this felt like the canonical page on "how do I compile solidity" which in turn generates ABI. Was also wondering if we've ever gotten this question with other networks ("how do I get ABI?"), seems like it would apply to any non-verified contract regardless of network

Reply:
fwiw, most users accessing these docs might already know how to compile an ABI, so I agree that this should be applicable to every other network, but doesn't matter since we've never had this question as far as I know.